src/js/entities-service/entities/patients.js

```diff
@@ -12,12 +12,11 @@ const _Model = BaseModel.extend({
   type: TYPE,
   urlRoot: '/api/patients',
   /* eslint-disable complexity */
-  validate({ first_name, last_name, birth_date, sex, _workspaces }) {
```

Comment:
My understanding is that this is only used when determining enabled/disabled state of the `Save` button when adding a new patient. I couldn't find any other place where this is used.